From 8cfe1d9005ae52bc69a71c3b94e23ac15acd13b7 Mon Sep 17 00:00:00 2001
From: Jason Wessel <jason.wessel@windriver.com>
Date: Tue, 23 Jun 2020 08:44:34 -0700
Subject: [PATCH] qemu-arm64: Defaults for booting with ostree

Set reasonable defaults for using booting with an ostree provided boot.scr.

Upstream-Status: Inappropriate [ ostree specific ]

Signed-off-by: Jason Wessel <jason.wessel@windriver.com>
---
 configs/qemu_arm64_defconfig | 2 ++
 include/configs/qemu-arm.h   | 4 ++++
 2 files changed, 6 insertions(+)

diff --git a/configs/qemu_arm64_defconfig b/configs/qemu_arm64_defconfig
index 084b4c1ca6..25aec77d43 100644
--- a/configs/qemu_arm64_defconfig
+++ b/configs/qemu_arm64_defconfig
@@ -46,3 +46,5 @@ CONFIG_USB=y
 CONFIG_DM_USB=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_EHCI_PCI=y
+CONFIG_MENU=y
+CONFIG_CMD_BOOTMENU=y
diff --git a/include/configs/qemu-arm.h b/include/configs/qemu-arm.h
index 1ef75a8783..ddb7d516e6 100644
--- a/include/configs/qemu-arm.h
+++ b/include/configs/qemu-arm.h
@@ -36,6 +36,10 @@
 	"fdt_high=0xffffffff\0" \
 	"initrd_high=0xffffffff\0" \
 	"fdt_addr=0x40000000\0" \
+	"console=ttyAMA0\0" \
+	"baudrate=115200\0" \
+	"loadaddr=0x40080000\0" \
+	"initrd_addr=0x43000000\0" \
 	"scriptaddr=0x40200000\0" \
 	"pxefile_addr_r=0x40300000\0" \
 	"kernel_addr_r=0x40400000\0" \
-- 
2.17.1

